core/server/utils/image-size-from-url.js

```diff
@@ -71,28 +74,27 @@ module.exports.getImageSizeFromUrl = function getImageSizeFromUrl(imagePath, tim
 
                         return resolve(imageObject);
                     } catch (err) {
-                        // @ToDo: add real error handling here as soon as we have error logging
+                        err.imagePath = imagePath;
```

Comment:
This won't work on LTS.
`err.context = imagePath`

And `logError(err, err.context)`.